Frontend/shared/src/functions/createGroceryItem/createGroceryItem.ts

```diff
@@ -1,4 +1,6 @@
-import { BASE_URL } from '@constants/config';
+import { BASE_URL } from '@constants';
+import { GroceryItem }from '@types';
+
 /*
```

Comment:
This comment can be removed because we can just look at the definition of a GroceryItem.